pom.xml

```diff
@@ -1943,11 +1943,11 @@
         <javassist.version>3.22.0-CR2</javassist.version>
         <javax.annotation.version>1.2</javax.annotation.version>
         <javax.annotation.bundle.version>1.2</javax.annotation.bundle.version>
-        <javax.el.version>2.2.4</javax.el.version>
-        <javax.el.impl.version>2.2.4</javax.el.impl.version>
+        <javax.el.version>3.0.1-b06</javax.el.version>
+        <javax.el.impl.version>3.0.1-b09</javax.el.impl.version>
```

Comment:
latest is b-10. but at the end we would need to update to some EPL licensed version before release

Reply:
I do not mean it is as a PR merge blocker, just a note of needs to be done before the 2.28 release which is going to happen very soon.